Test/Tests/PushChannelTests.swift

```diff
@@ -33,13 +36,13 @@ class PushChannelTests: XCTestCase {
 
     // RSH7a1
     func test__001__Push_Channel__subscribeDevice__should_fail_if_the_LocalDevice_doesn_t_have_an_deviceIdentityToken() {
-        waitUntil(timeout: testTimeout) { done in
+        waitUntil(timeout: testTimeout) { [userQueue] done in
             rest.channels.get(uniqueChannelName()).push.subscribeDevice { error in
                 guard let error = error else {
                     fail("Error is nil"); done(); return
                 }
                 expect(error.message).to(contain("cannot use device before device activation has finished"))
-                XCTAssertTrue(AblyTests.currentQueueLabel() == AblyTests.userQueue.label)
+                XCTAssertTrue(AblyTests.currentQueueLabel() == userQueue!.label)
```

Comment:
I guess it should work without "!" as well (here and below).

Reply:
It doesn’t — for some reason, when you capture an implicitly unwrapped optional in a capture list (i.e. the `[userQueue]` at the start of the closure) the compiler then starts to see it as an optional instead (try removing the `!` and you'll get a compiler error). Weird, right?